data/json/items/archery.json

```diff
@@ -571,7 +606,52 @@
       "dispersion" : 210,
       "durability" : 6,
       "clip_size" : 1,
-      "reload" : 70
+      "reload" : 70,
+      "valid_mod_locations": [["dampening", 1]]
+   },
+   {
+      "id" : "woodgreatbow",
+      "copy-from" : "longbow",
+      "override" : true,
+      "type" : "GUN",
+      "name" : "wooden greatbow",
+      "name_plural" : "wooden greatbows",
+      "description" : "An extremely large and stiff longbow, made with enormous limbs and a thick string to take an enormous amount of energy.  Takes incredible strength to draw.",
+      "price" : 150000,
+      "min_strength" : 18,
+      "weight" : 3200,
+      "volume" : 25,
+      "armor_data" : {
+         "covers" : ["TORSO"],
+         "coverage" : 7,
+         "material_thickness" : 2,
+         "encumbrance" : 20
```

Comment:
Encumbrance value of 20 is not changed compare to "longbow".

Reply:
Changed to 25, since it's bigger and heavier.